kms/api/Kms.Samples.Tests/CreateKeyLabelsTests.cs

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using Google.Cloud.Kms.V1;
+using Xunit;
+
+[Collection(nameof(KmsFixture))]
+public class CreateKeyLabelsTest
+{
+    private readonly KmsFixture _fixture;
+    private readonly CreateKeyLabelsSample _sample;
+
+    public CreateKeyLabelsTest(KmsFixture fixture)
+    {
+        _fixture = fixture;
+        _sample = new CreateKeyLabelsSample();
+    }
+
+    [Fact]
+    public void CreatesKey()
+    {
+        // Run the sample code.
+        var result = _sample.CreateKeyLabels(
```

Comment:
I just realized, you are not deleting what gets created on the samples that create something.
You can add methods in the fixture "MarkForDeletion" to which you pass the ID of what you created. Those are held in lists by the fixture and deleted on dispose.

Reply:
I am, but it's a bit indirect. I can document it if that'd be helpful. Everything gets created in the single KeyRing. The KeyRing has a dispose method that deletes all the things inside of it at the end.

I've run the samples a bunch of times now against my project and there's no left-over resources.

Reply:
Ah that's perfect, and true, I remember seeing that you were iterating over everything in the key ring. Yes, not a problem then.
Maybe add a note at the beginning of the test class, just to make it easy.